packages/zen-bull-vault/src/ZenBullStrategy.sol

```diff
@@ -1,6 +1,5 @@
-// SPDX-License-Identifier: GPL-3.0-only
+//SPDX-License-Identifier: BUSL-1.1
```

Comment:
`StrategyMath.sol` is imported here which is `//SPDX-License-Identifier: AGPL-3.0-only` , doesn't this mean we have to make it GPL?

It also has our contracts `VaultLib.sol` and `LeverageZen.sol` which are GPL2, and maybe need to be GPL (for LeverageZen)